demisto_sdk/scripts/validate_content_path.py

```diff
@@ -309,6 +316,14 @@ def _validate(path: Path) -> None:
             path.stem == path.parent.name and path.suffix in {".json", ".yml"}
         ):
             raise InvalidXDRCTemplatesFileName
+        elif first_level_folder == ContentType.MODELING_RULE.as_folder and not (
+            path.stem.startswith(path.parent.name)
+            and (
+                path.suffix in {".json", ".yml", ".xif"}
+                or path.stem.endswith("_schema")
```

Comment:
1. this is less strict than before - file path `Packs/bla/ModelingRules/bla/blaaaaa.json` will pass when it shouldn't
2. I think `_testdata.json` needs to be handled as well